docs/data_node/extract_node/tube.md

```diff
@@ -0,0 +1,70 @@
+---
+title: TubeMQ
+sidebar_position: 11
+---
+
+import {siteVariables} from '../../version';
+
+## Overview
+
+[Apache InLong TubeMQ](https://inlong.apache.org/docs/modules/tubemq/overview) is a distributed, open source pub-sub messaging and steaming platform for real-time workloads, trillions of massive data precipitation.
+
+## Version
+
+| Extract Node          | Version                                                      |
+| --------------------- | ------------------------------------------------------------ |
+| [TubeMQ](./tube.md) | [TubeMQ](https://inlong.apache.org/docs/next/modules/tubemq/overview): >=0.1.0<br/> |
+
+## Dependencies  
+
+In order to set up the `TubeMQ Extract Node`, the following provides dependency information for both projects using a
+build automation tool (such as Maven or SBT) and SQL Client with Sort Connectors JAR bundles.
+
+### Maven dependency
+
+<pre><code parentName="pre">
+{`<dependency>
```

Comment:
![image](https://user-images.githubusercontent.com/107393625/189801025-b5258040-3532-438b-99a7-26575cffe76d.png)
wrong format

Reply:
Please correct the above format errors  in the Chinese document in the same time.